src/pages/StudentsPage/StudentsPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import StudentsContainer from '../../containers/StudentsContainer/StudentsContainer';
+
+const StudentsPage = () => (
+  <div className="StudentsPage">
```

Comment:
классы не нужны, в Material-UI через useStyles  будут задаваться, ну и здесь никак не используются на данный момент
```suggestion
  <div>
```